sources/packages/api/src/route-controllers/education-program/models/save-education-program.dto.ts

```diff
@@ -8,9 +8,15 @@ import {
   getISODateOnlyString,
   getUserFullName,
 } from "../../../utilities";
+
 /**
- * Dto that represents education programs form object.
+ * Dto that represents education program with offering form object.
  */
+export interface EducationProgramWithOfferingsAPIOutDTO
+  extends EducationProgramDto {
```

Comment:
`EducationProgramDto` is a DTO used for input that, when refactored, will receive all the class decorators and all. I would suggest creating a specific DTO for the response. Maybe both classes can share the same interface.